explorer/interpreter/resolve_unformed.h

```diff
@@ -17,12 +15,16 @@ class FlowFacts {
  public:
   enum class ActionType {
     // Used at `VariableDefinition` with initialization.
+    // Adds a must-be-formed flow fact.
```

Comment:
I'm happy to have these comments, but can you swap the order? I view the "what this does" as more important than the "where it's used", so I'd put it first.

Reply:
Done.